pkgs/top-level/all-packages.nix

```diff
@@ -6130,6 +6130,8 @@ with pkgs;
   clangStdenv = if stdenv.cc.isClang then stdenv else lowPrio llvmPackages.stdenv;
   libcxxStdenv = if stdenv.hostPlatform.isDarwin then stdenv else lowPrio llvmPackages.libcxxStdenv;
 
+  codd = haskell.lib.compose.justStaticExecutables (haskellPackages.callPackage ../development/tools/database/codd { });
```

Comment:
Should be in pkgs/by-name

Reply:
Thanks. I followed the pkgs/by-name README and some other packages as examples. I hope I did the right thing.